File: LabsC#/Importante.cpp
using System;

class Program {
    static void Main(string[] args) {
        Console.Write("Ingrese un número entero: ");

        try {
            int numero = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Número ingresado: " + numero);
        } catch (FormatException) {
            Console.WriteLine("Error: Debe ingresar un número entero.");
        }
    }
}

File: LabsJava/Importante.java
import java.util.InputMismatchException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese un número entero: ");

        try {
            int numero = scanner.nextInt();
            System.out.println("Número ingresado: " + numero);
        } catch (InputMismatchException e) {
            System.out.println("Error: Debe ingresar un número entero.");
        }
    }
}
